### DISCUSSION TITLE
How to run a single unit test?

### DISCUSSION THREAD

**Discussion by knightXun** [openxla/xla#16329] [Q&A] [answered]
For example, if I want to run the case https://github.com/openxla/xla/blob/0fbf2e1eef1c45ade8ebc8e070b734d81112a30c/xla/service/algebraic_simplifier_test.cc#L119

what steps shoud i do?

**Comment by beckerhe** [comment, marked as answer]
`bazel test --test_filter=AlgebraicSimplifierTest.IsNonNegative_Op //xla/service:algebraic_simplifier_test` should do that for you.

**Reply by knightXun** [reply]
If I add some std::cout to print log, How can I see them when I run the test ?

**Reply by beckerhe** [reply]
If you add `--test_output=all` to the bazel command you should see the output.

**Reply by knightXun** [reply]
that's awesome!

**Reply by knightXun** [reply]
by the way, I ask some questions on discord of OpenXla , nobody answer.
